### DISCUSSION TITLE
Hosted Runners Not Able to Reach  http://archive.ubuntu.com/ubuntu

### DISCUSSION THREAD

**Discussion by papunsenapati07** [community/community#44171] [Actions]
### Select Topic Area

Bug

### Body

Hi We have been using hosted runners and all of a sudden we a experincing issues that the runners are not able to reach  http://archive.ubuntu.com/ubuntu, please find the screen shot below.

 Tried and tested in both ubuntu-20.04 and ubuntu-22.04
![image](https://user-images.githubusercontent.com/36064005/212220158-5c27a02d-d3a1-4a06-ac87-4477e8eb8853.png)

**Comment by regner** [comment]
Yeah this appears to be a larger issue with Ubuntu. We are having issues reaching the Ubuntu apt servers here in the UK.

**Reply by stuartford** [reply]
Do you have any links to other pages, issues, StackOverflow, status pages, tweets or anything about this? We are having similar issues in AWS eu-west-2.

**Reply by regner** [reply]
The issue just resolved itself a few minutes ago for us.

There was this post on ask ubuntu: https://askubuntu.com/questions/1450044/aws-ubuntu-22-04-jammy-ec2s-cant-apt-get-update-from-aws-archive-503s

Also my Twitter post: https://twitter.com/RegnerBA/status/1613866377923772416

Like I said though, issue seems to be resolved.

**Reply by regner** [reply]
Also to be clear when I say resolved, I mostly mean the connection timing out completely. Requests are still slow:
```
Fetched 25.0 MB in 2min 38s (158 kB/s)
```

**Reply by stuartford** [reply]
Thank you for the information. We've disabled any autoscaling until we are sure that we won't get any more stillborn servers!

**Reply by regner** [reply]
@stuartford don't know if this help or not, but download speeds have gone back to normal for us.

**Reply by gaborcsardi** [reply]
We are still seeing errors that are related I believe. This is from about 15 minutes ago: https://github.com/mlverse/torchaudio/actions/runs/3907792812/jobs/6685913889#step:3:137
```
  After this operation, 87.9 MB of additional disk space will be used.
  Get:1 http://azure.archive.ubuntu.com/ubuntu focal/main amd64 fonts-droid-fallback all 1:6.0.1r16-1.1 [1805 kB]
  Get:2 http://azure.archive.ubuntu.com/ubuntu focal/main amd64 poppler-data all 0.4.9-2 [1475 kB]
  Err:3 http://azure.archive.ubuntu.com/ubuntu focal/main amd64 dctrl-tools amd64 2.24-3
    Could not connect to azure.archive.ubuntu.com:80 (52.252.75.106), connection timed out [IP: 52.252.75.106 80]
  Err:4 http://azure.archive.ubuntu.com/ubuntu focal/main amd64 libfile-which-perl all 1.23-1
    Unable to connect to azure.archive.ubuntu.com:http: [IP: 52.252.75.106 80]
```

**Comment by lupusllc** [comment]
As of last night we were seeing very slow responses and timeouts in Kubernetes when doing anything with apt and apt-get in AWS and Azure for the West  US regions/zones, but not the East. This morning we're still seeing the some slowness, but so far no timeouts.

I also tested on my local Ubuntu VM going through the company VPN in the East and SW US, west was also exhibiting issues. East was not. Plus off the VPN in the NW I also saw issues.

So this appears to be region specific issues.

**Reply by lupusllc** [reply]
It's now only taking seconds for apt update, rather than minutes with possible timeouts in AWS, Azure and locally through VPN in the West. So it appears the issues are fully resolved as far we can tell.
